server/src/test/java/com/objectcomputing/checkins/services/agenda_item/AgendaItemServicesImplTest.java

```diff
@@ -1,331 +1,331 @@
-package com.objectcomputing.checkins.services.agenda_item;
+// package com.objectcomputing.checkins.services.agenda_item;
 
-import com.objectcomputing.checkins.services.checkins.CheckIn;
```

Comment:
delete this class if none of the tests are being used and make sure you have covered everything in service implementation via end to end tests.

Reply:
Deleted the file, covered in end to end tests.